src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,18 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import React from 'react';
+import { TodoInfo } from '../TodoInfo';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map(todo => (
+      <TodoInfo
+        key={todo.id}
+        todo={todo}
+      />
```

Comment:
```suggestion
      <TodoInfo key={todo.id} todo={todo} />
```
No need to move attributes to a new line if they are only two

Reply:
Agree

![image](https://user-images.githubusercontent.com/63306833/209950467-74737d98-42e0-4bb4-8c94-28c729fb2faf.png)